sql-plugin/src/main/scala/org/apache/spark/sql/rapids/GpuFileSourceScanExec.scala

```diff
@@ -420,6 +418,14 @@ case class GpuFileSourceScanExec(
           // Track the actual data size read from the file system, excluding data being pruned
           // by meta-level pruning.
           bf += "readBufferSize" -> createSizeMetric(DEBUG_LEVEL, "size of read buffer")
+          // This metric is used to post the time spent in generating the `skip_row` column
+          // in Delta Lake 3.3.0+
+          bf += "isRowDeletedColumnGenTime" ->
+            createNanoTimingMetric(ESSENTIAL_LEVEL, "time skiprow gen")
+          // This metric is used to post the time spent in generating the `row_index` column
+          // in Delta Lake 3.3.0+
+          bf += "rowIndexColumnGenTime" ->
```

Comment:
These metrics are delta-deletion-specific, not parquet. They should be gathered only when reading from delta tables in parquet.

Reply:
I think the metrics should be defined not here but in `GpuDelta33xParquetFileFormat`. We don't support it today, we may need to add some new interface to support it. Let's fix this in https://github.com/NVIDIA/spark-rapids/issues/13469.